hortimon/mothership/tickscripts/recreate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# use this script to recreate all the kapacitor alerts
+# docker-compose run kapacitor-cli
+# $ cd /usr/local/tickscripts
+# $ ./recreate.sh
+kapacitor delete tasks clone_temp_alert flower_temp_alert veg_temp_alert
```

Comment:
this script deletes kapacitor alerts, redefines our generic template, creates new alerts from our json files, and enables those alerts. make sure to run it from inside the `kapacitor-cli` container.

note that `./data/tickscripts` is mounted into the `kapacitor-cli` container at `/usr/local/tickscripts`